doc/doc-environment.yml

```diff
@@ -4,6 +4,7 @@ channels:
   - defaults
 dependencies:
   - pyne>=0.5.11=nomoab_noopenmc*
+  - openmc
```

Comment:
Just to confirm: we are not limited on openmc versions as a requirement? (I know this is the doc environment)

Reply:
Correct.